staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -142,6 +150,28 @@ func ObjectReaction(tracker ObjectTracker) ReactionFunc {
 			}
 			return true, nil, nil
 
+		// Check for Apply before Patch because Apply is made of Patch.
+		case ApplyActionImpl:
+			if action.GetSubresource() != "" {
+				return true, nil, errors.NewBadRequest(fmt.Sprintf("sub-resources are not supported yet."))
```

Comment:
IIUC, this needs to be required to adjust most of the tests, based on the list of failures: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/116855/pull-kubernetes-unit/1679166711774842880, for example this one: https://github.com/kubernetes/kubernetes/blob/3cbfa3e2cd7e756b1f51a5c7e014c415ecdb1253/pkg/controller/disruption/disruption_test.go#L1424. 

Is it planned to be covered in this PR, or a follow up?

Reply:
I think I have to fix the tests that broke within this PR.

Reply:
Faking some of the subresources isn't going to be easy though.